packages/react-docs/RELEASE-NOTES.md

```diff
@@ -56,21 +56,13 @@ section: developer-resources
 - **Tree view:** 
   - Fixed various bugs ([#5128](https://github.com/patternfly/patternfly-react/pull/5128))
 
-### Catalog view extension
-- Fixed typings in FilterSidePanelCategoryItem ([#5145](https://github.com/patternfly/patternfly-react/pull/5145))
-
-### Access console extension
-- Removed css imports ([#5147](https://github.com/patternfly/patternfly-react/pull/5147))
-- Replaced novnc-core usage with the original @novnc/novnc ([#5208](https://github.com/patternfly/patternfly-react/pull/5208))
```

Comment:
Just want to verify this shouldn't be in the notes?

Reply:
You are right!  I deleted one too many line.